javascript/ql/src/Security/CWE-730/ServerCrash.qhelp

```diff
@@ -42,15 +42,13 @@
 
 	<p>
 
-		The following server implementation checks if a client-provided
-		file path is valid and throws an exception if the check fails. It can
-		be seen that the exception is uncaught, and it is therefore reasonable to
-		expect the server to respond with an error response to client requests
-		that cause the check to fail.
-
-		But since the exception is uncaught in the context of an
-		asynchronous callback invocation (<code>fs.access(...)</code>), the
-		entire server will terminate instead.
+		The following server code checks if a client-provided file path is valid
+		before saving data to it. It would be reasonable to expect that the
+		server responds with an error response to client requests with invalid
+		file paths. However, the server instead throws an exception, which is
+		uncaught in the context of the asynchronous callback invocation
+		(<code>fs.access(...)</code>). This causes the entire server to
+		terminate abruptly.
```

Comment:
```suggestion
		before saving data to that path. It would be reasonable to expect that the
		server responds with an error in case the request contains an invalid
		file path. However, the server instead throws an exception, which is
		uncaught in the context of the asynchronous callback invocation
		(<code>fs.access(...)</code>). This causes the entire server to
		terminate abruptly.
```

Some suggestions as I I found this a bit hard to read

- "responds with an error response" sounds a bit repetitive.
- "client request" seems equivalent to just "request". In cases where we do use the term "client request" it's always about outgoing requests, as opposed to incoming requests.

Reply:
Thanks! I like those improvements.